### PR TITLE
Fixed bug where pulling media without remote id was not possible.

### LINKED ISSUE #209: 3.3.3 Regression
`divio project setup` always ends with this error now: `Error: Divio Cloud project file '.aldryn' could not be found! Please make sure you're in a Divio Cloud project folder and the file exists.`

### PR DESCRIPTION
**Pull request by chescales**
Fixed bug where pulling media without remote id was not possible.
Added explicit parameter names to avoid similar bugs in the future.
Closes #209

### CODE REVIEW

File: divio_cli/localdev/main.py [hunk 1]
```diff
@@ -387,10 +391,11 @@ def get_db_restore_command(self):
         return cmd.format(self.db_dump_path)
 
 
-def pull_media(client, stage, remote_id, path=None):
+def pull_media(client, stage, remote_id=None, path=None):
     project_home = utils.get_project_home(path)
     website_id = utils.get_aldryn_project_settings(project_home)['id']
     website_slug = utils.get_aldryn_project_settings(project_home)['slug']
+    remote_id = remote_id or website_id
```
Comment (glasnt) [inline divio_cli/localdev/main.py]: Was the the bug fix, and the rest is just adding explicit parameters?

File: divio_cli/localdev/main.py [hunk 2]
```diff
@@ -387,10 +391,11 @@ def get_db_restore_command(self):
...
     remote_project_name = (
         website_slug if remote_id == website_id
         else 'Project {}'.format(remote_id)
```
Comment (GaretJax) [inline divio_cli/localdev/main.py]: This is interesting... I had this yesterday, and I am quite sure the remote_project_name is not adapted if the remote_id is passed in explicitly when *pulling the database*.

File: divio_cli/localdev/main.py (hunk 1 above)
Comment (chescales) [reply divio_cli/localdev/main.py]: Yes indeed!

File: divio_cli/localdev/main.py (hunk 2 above)
Comment (chescales) [reply divio_cli/localdev/main.py]: Let me take a look, thanks for the notice

Comment (chescales) [reply divio_cli/localdev/main.py]: Well it should work indeed: https://github.com/chescales/divio-cli/blob/82423048e350badafd4bbfcee6f157d6e641eed8/divio_cli/localdev/main.py#L349-L352

**Comment by chescales** [comment]
@GaretJax @glasnt is there anything else before we can merge the PR? Cheers

File: divio_cli/localdev/main.py (hunk 1 above)
Comment (glasnt) [reply divio_cli/localdev/main.py]: It would be helpful if this line change was in a separate commit, but not mandatory.

Comment (chescales) [reply divio_cli/localdev/main.py]: Agree definitely 👍

**Pull request merged**